invenio_deposit/views/rest.py

```diff
@@ -220,6 +222,7 @@ def __init__(self, serializers, pid_type, ctx, *args, **kwargs):
         for key, value in ctx.items():
             setattr(self, key, value)
 
+    @require_api_auth()
```

Comment:
IMHO `@need_record_permission('read_permission_factory')` should be sufficient with latest updates on Invenio-REST.